internal/fs/inode/dir.go

```diff
@@ -850,18 +852,18 @@ func (d *dirInode) DeleteChildDir(
 			Generation: 0, // Delete the latest version of object named after dir.
 		})
 
-	if err != nil {
-		err = fmt.Errorf("DeleteObject: %w", err)
-		return
-	}
-
 	if d.bucket.BucketType() == gcs.Hierarchical {
+		// In Hierarchical bucket implicit object remains as a folder only.
```

Comment:
Update this comment to specify why err is ignored in hns case

Reply:
Done.